### PR TITLE
Add content to inspector

### PR DESCRIPTION
**Pull request by DannyS03**
Adds complementary technical content to multiformats throughout the inspector. Think it is reasonable to say that the inspector can generally be updated with enhanced UI/UX

cc. @lidel

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,32 +44,98 @@ <h1 class='f3 fw2 montserrat aqua ttu'>CID Inspector</h1>
         </a>
       </label>
       <pre class='f5 sans-serif fw5 ma0 pv2 dib overflow-x-auto w-100' id="hr-cid"></pre>
-      <div class='f5 sans-serif ma0 charcoal-muted ttu f7 tracked'>multibase - version -  multicodec - multihash (name : size : digest in hex)</div>
+      <div class='f5 sans-serif ma0 charcoal-muted ttu f6 tracked'>multibase - version -  multicodec - multihash (name : size : digest in hex)</div>
       <div class='dn' id="cid" ></div>
+      <span>
+        <p class='f5 sans-serif ma0 charcoal-muted ttu f7 tracked'>
+        </br> string: base (base)(cid-version - multicodec - multihash)
+        </br> binary: cid_version - multicodec - multihash
```
Comment (lidel) [inline index.html:52]: I think i know what you wanted to convey here, but description does not look intuitive. Perhaps human sentence is better here?

```suggestion
        </br>CID is encoded as String using Multibase.
        </br>After decoding Multibase, the remaining value is bytes. 
```

File: index.html [hunk 2]
```diff
@@ -44,32 +44,98 @@ <h1 class='f3 fw2 montserrat aqua ttu'>CID Inspector</h1>
...
     <div class='pt4'>
       <label class='db'>
         <a class='pb1 w-100 tracked ttu f6 teal-muted hover-aqua link' href="https://github.com/multiformats/multibase">
           Multibase
         </a>
+       </br>
       </label>
+      <p class='f5 sans-serif ma0 charcoal-muted fw4 pt2'>
+       </br>Self-describing data model for expressing raw binary
+       </br>data in a variety of base-encoding formats.
```
Comment (lidel) [inline index.html:65]: nit: we use "data model" only in context of https://ipld.io
https://docs.ipfs.tech/concepts/glossary/#multibase is a protocol for encoding stuff:
```suggestion
       </br>Self-describing protocol for expressing raw binary
       </br>data in a variety of base-encoding formats.
```

**Review by lidel — CHANGES_REQUESTED** [review]
Thanks!  This needs some UX  / devrel  :eyes:  


- Before: https://cid.ipfs.tech/#QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR
- This PR:  https://bafybeiekwowj7jkbguo77nf2tkqh76jkmnjyblwvyecn57tv6p6hcrcg3q.on.fleek.co/#QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR

@juliaxbow  @2color will you have time to provide feedback on this?

**Comment by juliaxbow** [comment]
Sure, will take a look and suggest a few changes. Some initial thoughts in the interim:

- Information hierarchy 
    - Unclear what is most important; hard to quickly find sections 
- Links
    - Links all take different formats; unclear what I can click or not
- Informational text
    - Great to link out to documentation but could be helpful to have more details on the page (or maybe once the link issue is fixed this won’t be as big of an issue)
    - The info text that exists is hard to differentiate from other content because of font choices

**Comment by juliaxbow** [comment]
Made some light suggestions for changes that should make this more readable to the average viewer without overhauling the page in its entirety, though if that's what you're looking for please let me know.

Update:
CID Inspector (recommendations)

With comments:
CID Inspector (marked up)

(poor quality) gif showing highlight on hover & link->section for CID components 
CID_explorer

**Comment by DannyS03** [comment]
> Made some light suggestions for changes that should make this more readable to the average viewer without overhauling the page in its entirety, though if that's what you're looking for please let me know.
> 
> Update: CID Inspector (recommendations)
> 
> With comments: CID Inspector (marked up)
> 
> (poor quality) gif showing highlight on hover & link->section for CID components CID_explorer [ CID_explorer ](https://user-images.githubusercontent.com/58958327/183345254-0d501795-4543-4d6b-9328-60733f2326d7.gif) [ ](https://user-images.githubusercontent.com/58958327/183345254-0d501795-4543-4d6b-9328-60733f2326d7.gif)

This is great

**Comment by hacdias** [comment]
I think there is a lot to unpack on this PR, as well as with the UX improvement suggestions.

**1)** I made some updates to the header based on @juliaxbow's suggestions. You can see the results here:

- https://bafybeiah2naqoecumkw3hsedru27oqr266k2sbaf7nc5xfk36sh4wumfpy.on.fleek.co/#QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR

@juliaxbow we have this thing (https://github.com/ipfs-shipyard/ipfs-css) with colors and some CSS that we use across different projects, including this one. I've applied your suggestions using the colors from `ipfs-css` and [`tachyons`](https://tachyons.io/).

**2)** I personally don't like the way this information is displayed with the `>`. I think that we should either have a link for the documentation for more in-depth details, or have a [folding thing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) with the information inside. 

image

Or maybe a "?" icon that, when pressed, shows more information.

Depending on how we solve this textual issue, we may, or may not, need the boxes around each zone because there wouldn't be that much information.

What do you all think?

**Review by hacdias — COMMENTED** [review]
I think there is a lot to unpack on this PR, as well as with the UX improvement suggestions.

**1)** I made some updates to the header based on @juliaxbow's suggestions. You can see the results here:

- https://bafybeiah2naqoecumkw3hsedru27oqr266k2sbaf7nc5xfk36sh4wumfpy.on.fleek.co/#QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR

@juliaxbow we have this thing (https://github.com/ipfs-shipyard/ipfs-css) with colors and some CSS that we use across different projects, including this one. I've applied your suggestions using the colors from `ipfs-css` and [`tachyons`](https://tachyons.io/).

**2)** I personally don't like the way this information is displayed with the `>`. I think that we should either have a link for the documentation for more in-depth details, or have a [folding thing](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) with the information inside. 

image

Or maybe a "?" icon that, when pressed, shows more information.

Depending on how we solve this textual issue, we may, or may not, need the boxes around each zone because there wouldn't be that much information.

What do you all think?

**Pull request closed without merging**